app/policies/assignment_policy.rb

```diff
@@ -28,6 +28,10 @@ def view_test_options?
     (check?(:run_tests?, role) && check?(:tests_enabled?)) || (role.student? && check?(:student_tests_enabled?))
   end
 
+  def stop_test?
```

Comment:
Good, but since this method corresponds to at least one action, it's not a 'helper policy', so should be moved above the comment on line 25